src/components/binding/binding-renderer.js

```diff
@@ -186,7 +192,8 @@ genotet.BindingRenderer.prototype.init = function() {
  * @private
  */
 genotet.BindingRenderer.prototype.getBindingRanges_ = function() {
-  if (this.data.overviewRangeChanged) {
+  if (this.data.overviewRangeChanged || !this.scaleSet_) {
```

Comment:
In loadFullTrack, set rangeChanged to true only when range changed. Do not set to false even when the range does not change. And here set rangeChanged to false when you update the scales.